### ISSUE TITLE
Concept

### ISSUE THREAD

**Issue by AlistairB** [scientific-defense-force/dear-it-professionals-site#1]
Premise: You (geeky person) can save the world!

Tl;dr: The world is in bad shape due to systemic reasons. Using technology and open source systems we can promote the problems and work on new a better systems to solve those problems. 

1. The world is in bad shape + getting worse
1.1 The environment
1.2 Wealth inequality

2. The problems are systemic
2.1 Capitalism
2.2 Globalization
2.3 Democracy (in the most powerful countries)

3. Thus the prevailing system needs to be changed.
3.1 Replaced with something new
3.2 Significantly adapted

4. This isn't happening
4.1 People in powerful democracies either aren't aware or don't care enough to push for change.
4.2 People believe that the problems are from 'corruption' not systemic issues.
4.3 People believe that democratic capitalism is the only viable system. New systems or heavy adaptions are not possible.
4.4 The far left believes in 'Marxism' but this has not shown to be a viable alternative and is not seriously considered by others.

5. Technology can be used to help with these problems
5.1 Websites / data analysis to promote the existence and severity of the current problems.
5.2 Websites / research can be conducted into alternative systems to solve the problems.

6. Crowd sourcing, open source, modern software development has shown that non-experts can contribute towards a complex goal
6.1 Alternative systems can be researched by non-experts.
6.2 There is a lot of possible data that can be exposed to the general public to help inform about these issues.
